projects/packages/image-cdn/tests/php/test_class.image_cdn.php

```diff
@@ -1463,7 +1463,6 @@ public function test_image_cdn_in_rest_response_with_edit_context() {
 
 	 * @author kraftbj
 	 * @requires PHPUnit 7.5
-	 * @covers Image_CDN::should_rest_image_cdn_image_downsize_insert_attachment
```

Comment:
If you want, it looks like the function is still named `should_rest_photon_image_downsize_insert_attachment`. Presumably a global search-and-replace changed the name incorrectly here.

OTOH, it looks like coverage is still recorded even in the absence of a `@covers` here, since we don't have `requireCoverageMetadata` set.